### ISSUE TITLE
Can we please get an answer on why this isn't working?  

### ISSUE THREAD

**Issue by OBAViJEST** [sultim-t/xash-rt#72]
![image](https://user-images.githubusercontent.com/12637059/221387639-15bfc6e3-4242-40a4-a7c5-4a8830c70f5a.png)

Why are unresolved issues being closed?

https://github.com/sultim-t/xash-rt/issues/36

https://github.com/sultim-t/xash-rt/issues/30

https://github.com/sultim-t/xash-rt/issues/2

No solutions anywhere

**Comment by sultim-t** [comment]
I'm out of ideas, more in-depth investigation is required with actual debugging tools.

There's some undefined behavior, I suppose.

**Comment by Tuxist** [comment]
![hl-rt_crash](https://user-images.githubusercontent.com/1216756/221416699-94125d41-b29d-4aee-b659-7092de996092.png)

i have intel arc 770 with two Xeon E5-2680v4 and Win 11 Enterprise.

**Comment by OBAViJEST** [comment]
> ![hl-rt_crash](https://user-images.githubusercontent.com/1216756/221416699-94125d41-b29d-4aee-b659-7092de996092.png)
> 
> i have intel arc 770 with two Xeon E5-2680v4 and Win 11 Enterprise.

Same thing on Win10 Pro 22H2 and GTX 1080

**Comment by kin37ik** [comment]
> > ![hl-rt_crash](https://user-images.githubusercontent.com/1216756/221416699-94125d41-b29d-4aee-b659-7092de996092.png)
> > i have intel arc 770 with two Xeon E5-2680v4 and Win 11 Enterprise.
> 
> Same thing on Win10 Pro 22H2 and GTX 1080

Nvidia GTX10XX Series cards dont support raytracing, you need at minimum a 20 series or above.

**Comment by xmb9** [comment]
> > > ![hl-rt_crash](https://user-images.githubusercontent.com/1216756/221416699-94125d41-b29d-4aee-b659-7092de996092.png)
> > > i have intel arc 770 with two Xeon E5-2680v4 and Win 11 Enterprise.
> > 
> > 
> > Same thing on Win10 Pro 22H2 and GTX 1080
> 
> Nvidia GTX10XX Series cards dont support raytracing, you need at minimum a 20 series or above.

incorrect, nvidia enabled raytracing support for pascal back in 2020. on my 1060 i can play both PrBoom-RT and Quake II RTX.
